raiden/tests/unit/test_raiden_event_handler.py

```diff
@@ -91,3 +105,181 @@ def detail_participants(_participant1, _participant2, _block_identifier, _channe
     RaidenEventHandler().on_raiden_event(
         raiden=raiden, chain_state=raiden.wal.state_manager.current_state, event=event
     )
+
+
+def test_pfs_handler_handle_routefailed_with_feedback_token():
```

Comment:
The following four tests you are adding have a very common setup phase. Can you perhaps abstract it in a function or use one of the factories we already got?

Reply:
Done